### PR TITLE
testing/squeezelite: new aport

### PR DESCRIPTION
**Pull request by trixpan**
This is a migration of a patch submitted via email over six months ago but apparently never merged / reviewed.

### CODE REVIEW

**Comment by trixpan** [comment]
note to self... never try to migrate a port from email again... :laughing:

**Comment by Cogitri** [comment]
Please use the usual commit msg standard used for aports :)

File: testing/squeezelite/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Contributor: Carl Chave <online@chave.us>, Andre F de Miranda <trixpan@users.noreply.github.com>
+# Maintainer:
```
Comment (Cogitri) [inline testing/squeezelite/APKBUILD]: No maintainer?

File: testing/squeezelite/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+pkgname=squeezelite
+pkgver=1.9.6.1196
+pkgrel=0
+pkgdesc="Lightweight headless squeezebox player for Logitech Media Server"
+url="https://github.com/ralph-irving/squeezelite"
+arch="all"
+license="GPL-3.0-or-later3"
+options="!check" # No test suite
+depends="flac alsa-lib faad2 mpg123 libvorbis libmad"
```
Comment (Cogitri) [inline testing/squeezelite/APKBUILD]: Pretty sure it autodetects these

File: testing/squeezelite/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+source="$pkgname-$pkgver.zip::https://github.com/ralph-irving/squeezelite/archive/2b508464dce2cbdb2a3089c58df2a6fbc36328c0.zip $pkgname.confd $pkgname.initd"
+builddir="$srcdir/squeezelite-2b508464dce2cbdb2a3089c58df2a6fbc36328c0/"
+
+build() {
+    cd "$builddir"
+    make
+}
+
+package() {
+    cd "$builddir"
```
Comment (Cogitri) [inline testing/squeezelite/APKBUILD]: same here

File: testing/squeezelite/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+options="!check" # No test suite
+depends="flac alsa-lib faad2 mpg123 libvorbis libmad"
+makedepends="flac-dev alsa-lib-dev faad2-dev mpg123-dev libvorbis-dev libmad-dev"
+install="$pkgname.pre-install $pkgname.post-install"
+subpackages="$pkgname-doc $pkgname-openrc"
+source="$pkgname-$pkgver.zip::https://github.com/ralph-irving/squeezelite/archive/2b508464dce2cbdb2a3089c58df2a6fbc36328c0.zip $pkgname.confd $pkgname.initd"
+builddir="$srcdir/squeezelite-2b508464dce2cbdb2a3089c58df2a6fbc36328c0/"
+
+build() {
+    cd "$builddir"
```
Comment (Cogitri) [inline testing/squeezelite/APKBUILD]: abuild does that automatically

File: testing/squeezelite/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+package() {
+    cd "$builddir"
+    install -Dm 755 squeezelite \
+        "$pkgdir"/usr/bin/squeezelite
+    install -Dm 644 doc/squeezelite.1 \
+        "$pkgdir"/usr/share/man/man1/squeezelite.1
+    install -Dm 644 "$srcdir"/squeezelite.confd \
+        "$pkgdir"/etc/conf.d/squeezelite
+    install -Dm 755 "$srcdir"/squeezelite.initd \
+        "$pkgdir"/etc/init.d/squeezelite
```
Comment (Cogitri) [inline testing/squeezelite/APKBUILD]: There's no install target for make?

**Comment by maxice8** [comment]
No answer for a month, closing.

**Pull request closed without merging**